tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/api/Environment.java

```diff
@@ -168,15 +229,51 @@ public BuildingList loadBuildings() {
   public BuildingList getBuildings() {
     return this.buildingList;
   }
+  
+  /**
+   * Builds a project on a piece of land.
+   * @param stakeholder The stakeholder id of the building project
+   * @param function The function to be used.
+   * @param floors The amount of floors for the building project.
+   * @param polygons The polygons describing the land.
+   */
+  public void build(Stakeholder stakeholder, Function function, int floors, String polygons) {
+    logger.debug("Building project started");
+    JSONArray parameters = new JSONArray();
+    parameters.put(stakeholder.getId());
+    parameters.put(function.getId());
+    parameters.put(floors);
+    parameters.put(polygons);
+    apiConnection.execute("event/PlayerEventType/BUILDING_PLAN_CONSTRUCTION/", 
+        CallType.POST, new StringResultHandler(), session);
+  }
+  
+  /**
+   * Buys a piece of land.
+   * @param buyerId The stakeholder id of the buyer.
+   * @param polygons The polygons describing the land.
+   * @param cost The amount of money per unit of land.
+   */
+  public void buyLand(int buyerId, String polygons, int cost) {
+    logger.debug("Buying land");
+    JSONArray parameters = new JSONArray();
```

Comment:
Consider refactoring this JSONArray into a dedicated object, see `Session.CloseSessionRequest`.